mama/c_cpp/src/gunittest/c/MainUnitTestC.cpp

```diff
@@ -60,16 +84,36 @@ static void parseCommandLine (int argc, char** argv)
 
     for (i = 1; i < argc;)
     {
-		if (strcmp ("-m", argv[i]) == 0)
-		{
-			gMiddleware = argv[i+1];
-			i += 2;
-		}
-		else if (strcmp ("-p", argv[i]) == 0)
-		{
-			gPayload = argv[i+1];
-			i += 2;
-		}
+        if (strcmp ("-m", argv[i]) == 0)
+        {
+            gMiddleware = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-p", argv[i]) == 0)
+        {
+            gPayload = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-tport", argv[i]) == 0)
```

Comment:
Do these parameters need to be run time configurable? The approach for unit tests to date has typically been for hard-coded values within the unit test itself to be used so that users don't have to memorize the correct command line to run the unit tests. The only parameters which (should) exist at the moment  are ones which control or are dependent on 'what' is tested as opposed to 'how' it's tested. I expect the global defaults you have already specified will be fine on their own.